qa-pipelines/tasks/cf-deploy.sh

```diff
@@ -49,11 +51,12 @@ helm install ${CAP_DIRECTORY}/helm/uaa/ \
 # Wait for UAA release
 wait_for_release uaa
 
-if [[ ${cap_platform} == "azure" ]] || [[ ${cap_platform} == "gke" ]]; then
+if [[ ${cap_platform} == "azure" ]] || [[ ${cap_platform} == "gke" ]] ||
+ [[ ${cap_platform} == "eks" ]]; then
     az_login
     azure_dns_clear
-    azure_wait_for_lbs_in_namespace uaa
-    azure_set_record_sets_for_namespace uaa
+    azure_wait_for_lbs_in_namespace uaa ${cap_platform}
```

Comment:
I don't think we need to pass in cap_platform as a parameter; the variable is already set in the common lib, which is sourced at the beginning of cf-deploy.sh